vcluster/_fragments/sync-from-host-configmap-example.mdx

```diff
@@ -0,0 +1,189 @@
+import Highlight from "@site/src/components/Highlight/Highlight";
+
+import Flow, { Step } from "@site/src/components/Flow";
+
+
+
+## From host config map sync example
+
+This guide shows how to sync k8s config map from host and use it in your workload inside the virtual cluster.
```

Comment:
```suggestion
This guide shows how to sync Kubernetes ConfigMaps from host clusters and how you can use them in your workload running inside virtual clusters.
```

Should we name Kubernetes resources uniformly in camelCase? Did we define that somewhere? (cc @Piotr1215)? If so, might make sense to rename all occurrences of "config map" to "ConfigMap" here. If not, just ignore my comment regarding this 😉

Reply:
@johannesfrey good call, we do have [guidelines](https://github.com/loft-sh/vcluster-docs/blob/aba0e93a803729fc32947088c21173a7f71a7554/CONTRIBUTING.md?plain=1#L51) as to how to capitalize Kubernetes objects. More details on [Kubernetes page](https://kubernetes.io/docs/contribute/style/style-guide/#documentation-formatting-standards).

@hidalgopl can you adjust accordingly?

Reply:
adjusted :+1: